android/app/src/main/java/com/onair/hearit/presentation/home/HomeViewModel.kt

```
@@ -40,6 +41,10 @@ class HomeViewModel(
     private val _toastMessage = SingleLiveData<Int>()
     val toastMessage: LiveData<Int> = _toastMessage
 
+    private lateinit var paging: Paging
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)